haskell/src/Monocle/Main.hs

```diff
@@ -75,7 +87,9 @@ run' port url configFile glLogger = do
       Warp.runSettings
         settings
         . cors (const $ Just policy)
-        . provideOptions monocleAPI
+        -- https://github.com/haskell-servant/servant/issues/672
+        -- TODO: commented out for now due to missing instances
+        -- . provideOptions monocleAPI
```

Comment:
```suggestion
```

If that doesn't work, I think we could add `corsIgnoreFailures = True` in the policy below. Though in my testing, the web development ui worked without provideOptions.

Reply:
Oh yeah, I've retried today an yes simply removing the provideOptions middleware solve the issue. (Maybe nginx was not started correctly on my setup)